packages/mobile/test/schemas.ts

```diff
@@ -312,7 +312,6 @@ export const v1Schema = {
       total: 0,
     },
     matchedContacts: {},
-    isValidRecipient: false,
```

Comment:
Looks like we need a migration to remove `isValidRecipient` and add `validationSuccessful` to existing `secureSendPhoneNumberMapping`.

Reply:
The issue is that we now have a `validationSuccessful` property for each phone number entry within the `secureSendPhoneNumberMapping` but we only had one universal `isValidRecipient`. In this case, what kind of migration is appropriate?

I wasn't sure so I just deleted `isValidRecipient` from the test schema and added `validationSuccessful` to the `SecureSendDetails` type.

Reply:
Yes this isn't going to break existing users.
If you decide not to add a migration, the type for `validationSuccessful` should be changed to `boolean | undefined` for correctness.